lib/pymedphys/docs/conf.py

```diff
@@ -39,7 +39,7 @@ def setup(app):
 # The short X.Y version
 # Currently being used to create mybinder link -- potentially need to rethink
 # this logic.
-version = "main"
+version = "fix-netlify"
```

Comment:
Need to revert this before merging

```suggestion
version = "main"
```